lib/solarwinds_apm/oboe_init_options.rb

```diff
@@ -181,7 +181,7 @@ def validate_token(token)
     end
 
     def validate_transform_service_name(service_name)
-      service_name = 'test_ssl_collector' if ENV['SW_APM_COLLECTOR'] =~ /java-collector:1222/
+      service_name = 'test_ssl_collector' if /java-collector:1222/.match?(ENV['SW_APM_COLLECTOR'])
```

Comment:
Hmm I question whether this line should actually be in the code, seems easy enough for testing setup to just specify a service name rather than pollute the code.

Reply:
I'd agree. I removed all the logic related with java-collector.